tallycoin_connect.js

```diff
@@ -142,7 +148,7 @@ function start_websocket(){
 	console.log('starting websocket');
 
 	clearInterval(wstimer);
-	ws = new WebSocket('wss://ws.tallycoin.app:8123/');
```

Comment:
Websocket server port must remain as 8123, since this is the Tallycoin server side, not client side.

Reply:
Oh yes, you are correct -- thanks for pointing that out. Fixed.